lib/SimpleSchema.js

```diff
@@ -591,6 +591,32 @@ class SimpleSchema {
     return def.label;
   }
 
+  /**
+   * Gets a field's property
+   *
+   * @param {String} [key] The schema key, specific or generic.
+   *   Omit this argument to get a dictionary of all labels.
+   * @param {String} [prop] Name of the property to get.
+   *
+   * @returns {any} The property value
+   */
+  get(key, prop) {
+    const def = this.getDefinition(key, ['type', prop]);
+
+    if (!def) return undefined;
+
+    if (schemaDefinitionOptions.includes(prop)) {
```

Comment:
@janowsiany versions are Node 4.8.2 & Meteor 1.4.4.1
There isn't a 'babel-polyfill' include, but it wasn't in your PR either, I don't think.  Did I miss it? I don't see it in the project dependencies, either.

This isn't my project and I don't have rights to do anything, so take my opinions for what they are (i.e. opinions with no weight).  In general, though, if it were my project, I would prefer an implementation that does not add an additional dependency vs one that does -- unless there is a compelling reason to add the dep.  With that said, is there something that Array.prototype.includes does that _.includes does not?  The underscore dependency is already in the project (and file) and, if it provides a comparable polyfill, why not use it? (Again, my comments and opinions are my own and have no overall bearing on the project.)